### ISSUE TITLE
Peer dependency failure (using Expo)

### ISSUE THREAD

**Issue by FritzHH** [pwcremin/react-native-bluemix#2]
I keep running into "unmet peer dependency" issues.  I am running this using the Expo install. Usually with the packages all I have to do is download them via NPM using npm install...<cmd>... and don't need to do the other Android or Swift linking (I am using the latest version of RN, I just started working with it a month ago).  I have attached a shot of the command line installation error log. 

![error](https://user-images.githubusercontent.com/17998327/32138205-dc422760-bbeb-11e7-86d7-b7dc9a4cb940.JPG)

![error2](https://user-images.githubusercontent.com/17998327/32138222-25825454-bbec-11e7-948b-7e4961ba40d2.JPG)

![error3](https://user-images.githubusercontent.com/17998327/32138314-bd14fa14-bbed-11e7-870c-df05714191f7.JPG)

**Comment by pwcremin** [comment]
I do not use Expo, but I am pretty sure it is pure javascript only, and you can't use native modules (like this project).   If you do not really need Expo, then create a project with react-native init <project name> and then you can try again.

**Comment by FritzHH** [comment]
Yes absolutely, I was using the create-react-native-app (being new at it), which you cannot do and also use native code.  Since then, so that I could learn what was happening behind the scenes,  I have wrote a native module for just speech-to-text using Bluemix using some functionality ideas I got from your module (thank you for that). 

The problem I was having, and for those new at this, the issue with the react-native-bluemix module is that IF you use react-native init <projectname> to create your project, it defaults to Android SDK 23.  However, 'com.ibm.watson.developer_cloud:android-sdk:0.4.2' needs minimum SDK 26 (as well as buildtools and appcompat versions to match).  So these need to be updated in multiple places in the app (manifest, build.gradle) to make the Bluemix go. 

Since I am new, I went through a lot of headache figuring out the vagaries of updating an app.  For other people new at it, just be prepared to delete the build folders and run gradlew clean in the android folder throughout the process if things go wrong.
